src/commands/features/create.ts

```diff
@@ -21,9 +23,67 @@ export default class CreateFeature extends CreateCommand {
         sdkVisibility: Flags.string({
             description: 'The visibility of the feature for the SDKs'
         }),
+        interactive: Flags.boolean({
+            char: 'i',
+            description: 'Interactive Flag Creation Mode'
```

Comment:
"Feature" would be more consistent with our terminology
```suggestion
            description: 'Interactive Feature Creation Mode'
```